### ISSUE TITLE
Schaltplan

### ISSUE THREAD

**Issue by micw** [jsphuebner/esp-egycounter#3]
Hallo Johannes,

ich bastel noch an meinem eigenen Lesekopf. Was hast du da vor dem ESP an Bauteilen? Gibt's davon einen Schaltplan?

Viele Grüße,
Michael.

**Comment by jsphuebner** [comment]
Malst du mir einen? ;)
"We then pass the signal to some signal conditioning circuitry. The circuit is supplied by 5V and consists of a 15k pull-up resistor (it took some scoping to find the best value), a 74ACT04 inverter TTL chip (because parts bin), a 470 ohm series resistor, a voltage limiting diode towards 3.3V, a 3.3V linear LDO, and finally an ESP8266 wifi module, doesn't really matter which flavour."

Fototransistor -> Pull-Up 15k -> TTL gate -> 470 Ohm -> ESP Rx

**Comment by micw** [comment]
Hoppla, das ist ganz schön viel Material ;-) Wenn du ausschließlich einen ESP8266 dranhängen willst, brauchst du nur nen Fototransistor, und nen Pullup-Widerstand und kannst das mit 3.3V laufen lassen. Inverter ist auch nicht nötig, man kann den Pin per Software invertieren.

Ich habe heute meinen Schaltplan in alter und neuer Version mal als Projekt verpackt: https://github.com/micw/ir-lesekopf

**Comment by jsphuebner** [comment]
Ja, funktioniert wohl auch. Ich glaube ich hatte ein paar Bitfehler mit dieser Variante. Der Inverter invertiert 2x :) Ich nutze ihn nur als Schmitt-Trigger

**Comment by micw** [comment]
Alles klar. Hatte auch überlegt, ob ich den Schmitt-Trigger noch mit einbaue. Aber am Oszi sieht das Signal schon _sehr_ sauber auch nach dem Transistor, so dass ich drauf verzichtet habe.
